modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="enabling-assisted-installer-service-on-bare-metal_{context}"]
+
+.Enabling assisted installer service on bare metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. {rh-rhacm-first} ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the {rh-rhacm} hub cluster.
+
+For distributed units (DUs), {rh-rhacm} supports single node OpenShift deployments, which run on a single bare metal host. The single node cluster acts as both a control plane and a worker node. {rh-rhacm} supports single node OpenShift deployments for {product-title} on top of a single BMO DU.
+
+.Enable assisted installer service
+
+.Prerequisites
+
+* Install {product-title} {product-verstion} on a hub cluster.
+* Install {rh-rhacm} and create the MultiClusterHub resource.
+* Create PersistentVolume custom resources (CR) for database and file system storage.
+* Install openshift-install CLI for the desired version of {product-title}.
```

Comment:
What is the `openshift-install` CLI? Do you mean the `oc` CLI? If so:

```suggestion
* You have installed the OpenShift CLI (`oc`).
```

In either case:

> 🚫  desired adjective
Do not use. Use "required" or "the that you want".

Reply:
Updated as suggested.